File: Java Programming/Competitive Programming/Coding Challenges 10/program3.java
//Q3.Write a program which accept number from user and print even factors of that number.
//Input : 36
//Output: 2 6 12 18

import java.util.Scanner;

class program3
{
    static void DisplayEvenFactor(int iNo)
    {
        int i = 0;

        if (iNo <= 0)
        {
            iNo = -iNo;
        }

        for (i = 1; i <= iNo / 2; i++)
        {
            if ((iNo % i == 0) && (i % 2 == 0) && (i != 4))
            {
                System.out.print(i + "\t");
            }
        }
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter number");
        iValue = sobj.nextInt();

        DisplayEvenFactor(iValue);

        sobj.close();
    }
}

File: C++ Programming/Competitive Programming/Coding Challenges 8/program3.cpp
//3.Write a program which accept number from user and display all its non factors.
//Input : 12
//Output : 5 7 8 9 10 11
//Input : 13
//Output : 2 3 4 5 6 7 8 9 10 11 12
//Input : 10
//Output : 3 4 6 7 8 9

#include <iostream>
using namespace std;

void NonFact(int iNo)
{
    int i = 1;

    if (iNo < 0)
    {
        iNo = -iNo;
    }

    while (i < iNo)
    {
        if (iNo % i == 0)
        {
            i++;
            continue;
        }

        cout << i << "\t";
        i++;
    }
}

int main()
{
    int iValue = 0;

    cout << "Enter number ";
    cin >> iValue;

    NonFact(iValue);

    return 0;
}